### ISSUE TITLE
Crash on joining server (ATM Expert)

### ISSUE THREAD

**Issue by raphydaphy** [whatthedrunk/allthemods#253]
**Issue Description:** When I tried to join the ATM Expert test server, 158.69.212.150:25565, I instantly crashed 😢 

**What happens:** I crash to the launcher, heres the crash report: http://pastebin.com/C9esyFuM

**What you expected to happen**: I'd join the server and all would be good 👍 

**Steps to reproduce:**
1. Join (any?) ATM Expert server, I was joining the official test server 158.69.212.150:25565
2. Crash.. :(

**Version of All the Mods:** ATM Expert v0.62 (Beta) (Latest)

**Added mods (Optifine, Fastcraft, etc.):** Didn't add any

**Comment by al132** [comment]
This is one of the crashes that we've had on and off when testing the pack on the server, and unfortunately I have no idea what it is yet.

**Comment by queso1307** [comment]
I joined the test server and played fo a little bit. I disconnected normally and wanted to reconnect after restarting Minecraft because of sound issues and the game keeps crashing. I can see the crosshair in a completely black world an shortly after the screen resets to the usual dirt texture background and the Game crashes every time i try to connect to you test server.

**Crash report**:  http://pastebin.com/ms21PFyJ

**Version of All the Mods**: ATM Expert v0.62 (Beta) (Latest)
**Added mods (Optifine, Fastcraft, etc.)**: Didn't add any
